microsoft/testsuites/power/common.py

```diff
@@ -133,11 +166,11 @@ def run_network_workload(environment: Environment) -> Decimal:
 
 
 def cleanup_env(environment: Environment) -> None:
-    remote_node = cast(RemoteNode, environment.nodes[0])
-    startstop = remote_node.features[StartStop]
-    startstop.start()
-    for node in environment.nodes.list():
-        kill = node.tools[Kill]
-        kill.by_name("iperf3")
-        kill.by_name("fio")
-        kill.by_name("stress-ng")
+    try:
+        for node in environment.nodes.list():
+            kill = node.tools[Kill]
+            kill.by_name("iperf3")
+            kill.by_name("fio")
+            kill.by_name("stress-ng")
+    finally:
```

Comment:
finally/pass do nothing. If it needs to eat the exception, it should be an `except`.